server/src/setup/http/express/express.ts

```diff
@@ -0,0 +1,7 @@
+import { APIRouter } from './api-router'
+import { App } from './app'
+
+export interface Express {
+  apiRouter: APIRouter
+  app: App
+}
```

Comment:
Is having this type in it's own file right now worth the cognitive load? Can we move this into the file `./waterpark-express.ts`?

Reply:
Hm, this was something I was also confused about while trying to set the files up. I put the type and the factory in the same file if there was only one sensible way to construct the type. In theory, we have infinitely many ways to create an `Express` object. The way we set up Waterpark is an arbitrary one of those ways.

As you pointed out, that seems to be leading to more confusion than it's worth. I'll put them in the same file to signify that it's the only way that _we_ will create `Express` objects